prow/config/config.go

```diff
@@ -925,6 +939,13 @@ func parseProwConfig(c *Config) error {
 		c.PodNamespace = "default"
 	}
 
+	if c.GitHubOptions.LinkURL == "" {
+		c.GitHubOptions.LinkURL = "https://github.com"
+	}
+	if _, err := url.Parse(c.GitHubOptions.LinkURL); err != nil {
```

Comment:
Store the value.

Reply:
Can I have an example of this in the code base, please?

Reply:
We do this for regex on config load, for URLs on ghproxy options load, @cjwagner is doing it for the GCS object in his new PR for Tide here: https://github.com/kubernetes/test-infra/pull/11016/files

Reply:
We just want a field on the options that holds the parsed value

Reply:
Take a look at these commits, I think that is what you were asking for.